internal/pkg/errors/strings.go

```diff
@@ -118,14 +118,10 @@ const (
 	InvalidChoiceMsg = "%s is not a valid choice"
 
 	// General
-	CreatedResourceMsg            = "Created %s \"%s\".\n"
-	DeletedResourceMsg            = "Deleted %s \"%s\".\n"
-	DeleteResourceConfirmMsg      = "Are you sure you want to delete %s \"%s\"?\nTo confirm, type \"%s\". To cancel, press Ctrl-C"
-	DeleteResourceConfirmYesNoMsg = `Are you sure you want to delete %s "%s"?`
-	DeleteACLConfirmMsg           = "Are you sure you want to delete the ACL corresponding to these parameters?"
-	DeleteACLsConfirmMsg          = "Are you sure you want to delete the ACLs corresponding to these parameters?"
-	RequestedDeleteResourceMsg    = "Requested to delete %s \"%s\".\n"
-	UpdatedResourceMsg            = "Updated %s \"%s\".\n"
+	CreatedResourceMsg       = "Created %s \"%s\".\n"
+	DeletedResourceMsg       = "Deleted %s \"%s\".\n"
+	DeleteResourceConfirmMsg = "Are you sure you want to delete %s \"%s\"?\nTo confirm, type \"%s\". To cancel, press Ctrl-C"
```

Comment:
i remember some resources are like api-key `id`, some are cluster `name`. has it changed?

Reply:
They still work that way for single deletion, because I still want it to match the UI flow. But for multiple ids, all of them switch to a `y/n` prompt.